docs/extensions-plugins/obsidian.mdx

```diff
@@ -0,0 +1,186 @@
+    >
+      <div style={{ maxWidth: "fit-content", minWidth: "fit-content" }}>
+        <p
+          style={{
+            padding: "8px 20px",
+            borderRadius: "8px",
+            border: "1px solid #f3f5f6",
+            color: "white",
+            fontWeight: "500",
+            background: "black",
+            marginBottom: "8px",
+            marginTop: "3px",
+            marginRight: "16px",
+          }}
+        >
+          {props.children}
+        </p>
+      </div>
+    </a>
+  );
+};
+
+export const ReleasePill = (props) => {
+  return (
+    <a
+      style={{ textDecoration: "none" }}
+      target="_blank"
+      href={"https://code.pieces.app/updates"}
+    >
+      <div style={{ maxWidth: "fit-content", minWidth: "fit-content" }}>
+        <p
+          class="dark:text-white text-black"
+          style={{
+            padding: "8px 20px",
+            borderRadius: "8px",
+            border: "1px solid #f3f5f6",
+            backgroundColor: "transparent",
+            fontWeight: "500",
+            marginBottom: "8px",
+            marginTop: "3px",
+            marginRight: "16px",
+          }}
+        >
+          {props.children}
+        </p>
+      </div>
+    </a>
+  );
+};
+
+export const GridFlexRow = (props) => {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "start",
+        flexWrap: "wrap",
+      }}
+    >
+      {props.children}
+    </div>
+  );
+};
+
+export const HeaderLogoImage = (props) => {
+  return (
+    <img
+      style={{ marginLeft: "0px", height: "30%", marginRight: "12px" }}
+      src={"/assets/obsidian.png"}
+    />
+  );
+};
+
+<GridFlexRow>
+  <h1>Pieces Obsidian Plugin</h1>
+</GridFlexRow>
+
+<br></br>
+
+<GridFlexRow>
+  <InstallButton>Get the Obsidian Plugin</InstallButton>
+  <ReleasePill children="Current Version: {{ versions.obsidian }}"></ReleasePill>
+</GridFlexRow>
+
+<br></br>
+
+# Your Guide to Getting Started with Pieces for Obsidian
+
+_Pieces for Obsidian Pieces captures **NO identifiable user data**. Our **Local only** architecture means your notes never have to leave your device!_
+
+_The <a target="_blank" href="{{ links.website.pfd_desktop_install }}">Pieces Suite</a> must be downloaded and installed separately from the Obsidian Plugin in order for the Plugin to work properly._
+
+## Save your Snippets
+
+- To save a snippet, highlight the text, right-click, and select "Save to Pieces."
+  **OR**
+- Click the Pieces button within any code block.
+  **OR**
+- Highlight the desired code and use our dedicated keyboard shortcut
+  `Ctrl+Shift+P` (\[⌘⇧P\] on macOS)
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_RIGHTCLICK.gif" />
+
+## Find your Snippets
+
+Open the Pieces View from the Ribbon Menu.
+
+- View by **Recent** and by **Language**
+- Search your snippets to save time searching through your markdown.
+
+<Image zoom src="./assets/obsidian/OBSIDIAN_SEARCH.gif" />
```

Comment:
adding alt="" text - you can also remove the proceeding ./assets in your filenames